test/ergw_http_api_SUITE.erl

```diff
@@ -127,6 +148,109 @@ end_per_suite(Config) ->
     ok = lib_end_per_suite(Config),
     ok.
 
+http_api_session_info() ->
+    [{doc, "Check /session/id/bearer_id API"}].
+http_api_session_info(Config) ->
+    S = make_gtp_socket(Config),
+    {GtpC1, _, _} = create_pdp_context(S),
+
+    URL1 = get_test_url("/api/v1/sessions/111111111111111"),
+    {ok, {_, _, Body1}} = httpc:request(get, {URL1, []},
+                                        [], [{body_format, binary}]),
+    Res1 = jsx:decode(Body1, [return_maps]),
+    Session1 = maps:get(<<"Session">>, Res1),
+    ?match(#{<<"MSISDN">> := ?MSISDN, <<"IMSI">> := ?IMSI}, Session1),
+
+    URL2 = get_test_url("/api/v1/sessions/111111111111112"),
+    {ok, {{_, StatusCode, _}, _, Body2}} = httpc:request(get, {URL2, []},
+                                                         [], [{body_format, binary}]),
+    Res2 = jsx:decode(Body2, [return_maps]),
+    ?match(#{<<"Session">> := []}, Res2),
+    ?match(404, StatusCode),
```

Comment:
the StatusCode check could use ```?equal``` and go before the jsx:decode

Reply:
fixed